### PR TITLE
Fix structure of volume-coupled-diffusion

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
See #461

### CODE REVIEW

File: volume-coupled-diffusion/drain-fenics/run.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -e -u
+
+python3 ../solver-fenics/volume-coupled-diffusion.py --drain
```
Comment (MakisH) [inline volume-coupled-diffusion/drain-fenics/run.sh:4]: We need to merge back from develop once #479 is merged (or merge this beforehand).

File: volume-coupled-diffusion/solver-fenics/output/.gitkeep
Comment (MakisH) [inline volume-coupled-diffusion/solver-fenics/output/.gitkeep:1]: Is this still needed for the `solver-fenics` directory? Should it be at the `source-fenics`, instead?

File: volume-coupled-diffusion/drain-fenics/clean.sh
Comment (MakisH) [inline volume-coupled-diffusion/drain-fenics/clean.sh:1]: This cleaning script deletes also the `output/.gitkeep`, which then shows up in the `git status`.

I have the impression we don't need these files at all.

Comment (BenjaminRodenberg) [reply volume-coupled-diffusion/drain-fenics/clean.sh:1]: Done. Good point.

File: volume-coupled-diffusion/solver-fenics/output/.gitkeep
Comment (BenjaminRodenberg) [reply volume-coupled-diffusion/solver-fenics/output/.gitkeep:1]: Done. Good point.

File: volume-coupled-diffusion/drain-fenics/clean.sh
Comment (BenjaminRodenberg) [reply volume-coupled-diffusion/drain-fenics/clean.sh:1]: True. Removed them.

File: volume-coupled-diffusion/drain-fenics/run.sh (hunk 1 above)
Comment (MakisH) [reply volume-coupled-diffusion/drain-fenics/run.sh:4]: I will do that in my branch.

**Review by MakisH — APPROVED** [review]

**Pull request merged**